### ISSUE TITLE
Accessibility - Most Issues Will Apply for Future Android Application

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Ilab-do-good#43]
Most of these issues will apply for the Android application too.

**Comment by adriennerlong** [comment]
Have you been able to test the Android version and identify additional issues? If the Android app is not ready to launch on September 19, we will do a later release. Our main priority for the September 19 launch is the IOS app.

**Comment by jfc3-dol** [comment]
@adriennerlong,

I have note seen that the application is available to build and place on our Android device.

@giridhar,

Have you seen the Android code for ComplyChain?

**Comment by carolynhuang** [comment]
@jfc3-dol @giridhar The APK is called do-good-new.apk and should be located here:
https://github.com/USDepartmentofLabor/Ilab-do-good/tree/master/releases_for_testing/android

Please let us know if you are still unable to access it.

**Comment by jfc3-dol** [comment]
Once the Android application is built and deployed to our Android device I will begin testing.

**Comment by jfc3-dol** [comment]
@adriennerlong,

Start testing the Android device on Friday after finishing my testing of the iOS version.

**Comment by adriennerlong** [comment]
#43: Fixed.  All changes are mirrored on all platforms.

**Comment by jfc3-dol** [comment]
Going to close this issue because we tagged and the issues that apply to more than one platform using different labels.
